### PR TITLE
Add code of conduct to repository

### PR DESCRIPTION
**Pull request by ascott1**
This introduces the official CFPB code of conduct policy to the repo.

## Additions

- `CODE_OF_CONDUCT.md`

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)

### CODE REVIEW

**Review by Scotchester — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+## Credits
+
+The CFPB is greatly appreciative of the multiple sources that we drew from to build this Code of Conduct, including:
+
+- [The Technology Transformation Services (TTS) Code of Conduct](https://18f.gsa.gov/code-of-conduct/)
+- [The Contributor Covenant](https://www.contributor-covenant.org/)
+- [Code for America Code of Conduct](https://github.com/codeforamerica/codeofconduct)
+- [Ada Initiative: HOWTO design a code of conduct for your community](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/)
+- [Geek Feminism Code of Conduct](https://geekfeminismdotorg.wordpress.com/about/code-of-conduct/)
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:115]: TRIVIAL: maybe add an ending newline. I think we have this for other markdown files in the repo.

**Review by willbarton — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+## Scope
+
+We expect everyone on the CFPB team, and those contributing to our open source community, to exhibit 
+these behaviors and abide by applicable federal laws and CFPB policies. In addition, we expect 
+everyone within CFPB spaces to exhibit these behaviors and refrain from behavior prohibited by 
+anti-harassment laws and federal policies on harassment. These spaces include:
+
+- CFPB’s physical offices,
+- CFPB events and meetings,
+- All of CFPB’s online forums and virtual collaboration tools, including code repositories
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:22]: Bulleted list should either omit trailing commas, end in periods, or have a trailing period for the last item.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- National origin
+- Disability
+- Age (40 years or older)
+- Genetic information
+- Parental status
+- Political affiliation
+- Marital status
+- Uniformed status
+- Membership in a labor organization or union activities
+- Prior equal employment opportunity (EEO) or whistleblower activity, or
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:60]: Should trailing `or` have an ellipsis?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+If you are a CFPB employee, former CFPB employee, or job applicant to CFPB and believe you have been 
+discriminated against or harassed on the basis of race, color, religion, sex (including pregnancy, 
+sexual orientation, transgender status, gender identity or expression, gender non-conformity, or sex 
+stereotyping of any kind), national origin, disability, age (40 years or older), genetic information, 
+parental status, or retaliated against for prior Equal Employment Opportunity (EEO) activity, contact the CFPB’s Office of Civil Rights.
+
+CFPB_EEO@consumerfinance.gov
+
+(202) 435-9EEO
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:85]: Can we put the actual number for `435-9EEO` next to this?

**Comment by ascott1** [comment]
@anselmbradford I'm going to make your suggested formatting edits in [the primary file](https://github.com/cfpb/open-source-project-template/blob/master/CODE_OF_CONDUCT.md)

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+(855) 233-0362
+TTY: (202) 435-9742
+
+Office of Civil Rights
+Consumer Financial Protection Bureau
+1700 G Street, NW
+Washington, D.C. 20552
+
+For help filing a complaint about discrimination on the basis of marital status, political 
+affiliation, or any other non-merit factor, or for claims of retaliation for [whistleblower activity](https://www.consumerfinance.gov/office-civil-rights/whistleblowers/), contact the [Office of Special Counsel](http://www.osc.gov/) or the [Merit Systems Protection Board](http://www.mspb.gov/).
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:95]: Two `http` addresses could be `https`

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+For help filing a complaint about discrimination on the basis of uniformed status, you may contact 
+the [Veterans’ Employment and Training Service (VETS)](http://www.dol.gov/vets/) at the Department of Labor, the [Merit Systems Protection Board](http://www.mspb.gov/), or the [Office of Special Counsel](https://osc.gov/), depending on the circumstances.
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:98]: `http` addresses could be `https`

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (ascott1) [reply CODE_OF_CONDUCT.md:85]: This matches the information on our website

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+For help filing a complaint about discrimination on the basis of membership in a labor organization, 
+you may contact the [Federal Labor Relations Authority](http://flra.gov/) or your union (if applicable).
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:101]: `http` could be `https`

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+### Equal employment opportunity policy
+
+For more information about the CFPB’s equal employment opportunity (EEO) policies and procedures visit https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/
```
Comment (anselmbradford) [inline CODE_OF_CONDUCT.md:105]: Make https://www.consumerfinance.gov/office-civil-rights/eeo-policy-and-reports/ a link.

**Review by anselmbradford — CHANGES_REQUESTED** [review]
Minor fixes listed above

**Review by anselmbradford — APPROVED** [review]
> @anselmbradford I'm going to make your suggested formatting edits in the primary file

Ah cool, I'll approve here then and trust it'll get sorted.

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (anselmbradford) [reply CODE_OF_CONDUCT.md:85]: Can we update this on the website?

**Comment by ascott1** [comment]
Thanks for the review @anselmbradford! I made the majority of your suggestions in the primary repo and have updated them here as well.

**Pull request merged**